app/models/user.rb

```diff
@@ -15,4 +18,12 @@ def normalize_queue_item_positions
   def queued_video?(video)
     queue_items.map(&:video).include?(video)
   end
+
+  def follows?(another_user)
+   following_relationships.map(&:leader).include?(another_user) 
```

Comment:
Another way to do this is to use `pluck`:

```ruby
following_relationships.pluck(:leader_id).include?(another_user.id)
```

This way doesn't pull all of the relationships into memory at once.